ostelco-ios-client/ViewControllers/ESim/ESIMPendingDownloadViewController.swift

```diff
@@ -113,4 +111,20 @@ class ESIMPendingDownloadViewController: UIViewController {
             }
         }
     }
+    
+    override func viewDidLoad() {
+        if let region = OnBoardingManager.sharedInstance.region {
+            getSimProfileForRegion(region: region)
+        } else {
+            #warning("RegionResponse contains simProfile so we don't need to fetch the simProfiles for region in this case.")
```

Comment:
I'll fix this right now and push and update to this PR

Reply:
ok

Reply:
fixed, this wasn't an issue, the getSimProfileForRegion already checks region for a simProfile and only calls get /simProfiles if region does not contain any